adding details on burning HNT for DCs to maker wallets (#146)
helium/docs: docs/mine-hnt/hotspot-makers/burn-hnt-to-maker-wallet.mdx

---
id: burn-hnt-to-maker-wallet
hide_title: true
sidebar_label: Burning HNT to Maker Wallet
slug: /mine-hnt/hotspot-makers/burn-hnt-to-maker-wallet
---

import useBaseUrl from "@docusaurus/useBaseUrl";

# Burning HNT to Your Maker Wallet

> Any [approved Hotspot Maker](/mine-hnt/approved-makers) is responsible for supplying Data Credits (DCs) to the onboarding server for their Hotspots. They do this by burning HNT to their 

Reviewer: ```suggestion
Any [approved Hotspot Maker](/mine-hnt/approved-makers) is responsible for supplying Data Credits (DCs) to the onboarding server for Hotspots they sell. They do this by burning HNT to their 
```

specific maker address. 

:::info 

> A full list of approved maker wallet addresses [can be seen via this API](https://onboarding.dewi.org/api/v2/makers).

Reviewer: Can we automatically generate a table with this API response?

Author: In browser? Or in the docs?

:::

## Maker Wallet Address Integration

> As an approved Maker, you've been supplying the DeWi with the onboarding details for all the Hotspots you manufacture and distribute. In turn, the DeWi adds them to the onboarding server. When a Hotspot is onboarded by your end customer, the onboarding server knows to associate it with your Maker ID, and associated Maker Wallet address. If, when a Hotspot is onboarded, it's present in the onboarding server and you have ample Data Credits to pay the onboarding fees, the blockchain will add the Hotspot. And if you've [integrated your Hotspot in the Helium Mobile Wallet onboarding flow](/mine-hnt/hotspot-makers/hotspot-integration-testing), the Wallet knows to use your approved Maker Wallet address. At this point, the Data Credits will be flushed from your wallet address. 

Reviewer 2: As an approved maker, you **will supply DeWi the onboarding** details...

Reviewer 2: > And if you've integrated your Hotspot in the Helium Mobile Wallet onboarding flow

Change to "And if you've integrated your Hotspot in the Hotspot App onboarding flow

Reviewer 2: > At this point, the Data Credits will be flushed from your wallet address. 

Change to "At this point, Data Credits will be consumed by the onboarding transactions (Add Hotspot and/or Assert Location) and deducted from your Maker wallet address.

Author: Fixed.

> As an approved Maker, you've been supplying the DeWi with the onboarding details for all the Hotspots you manufacture and distribute. In turn, the DeWi adds them to the onboarding server. When a Hotspot is onboarded by your end customer, the onboarding server knows to associate it with your Maker ID, and associated Maker Wallet address. If, when a Hotspot is onboarded, it's present in the onboarding server and you have ample Data Credits to pay the onboarding fees, the blockchain will add the Hotspot. And if you've [integrated your Hotspot in the Helium Mobile Wallet onboarding flow](/mine-hnt/hotspot-makers/hotspot-integration-testing), the Wallet knows to use your approved Maker Wallet address. At this point, the Data Credits will be flushed from your wallet address. 

Reviewer: ```suggestion
As an approved Maker, you've been supplying the DeWi with the onboarding details for all the Hotspots you manufacture and distribute. In turn, the DeWi adds them to the onboarding server. When a Hotspot is onboarded by your end customer, the onboarding server knows to associate it with your Maker ID, and associated Maker Wallet address. If, when a Hotspot is onboarded, it's present in the onboarding server and you have ample Data Credits to pay the onboarding fees, the blockchain will add the Hotspot. And if you've [integrated your Hotspot in the Helium Mobile Wallet onboarding flow](/mine-hnt/hotspot-makers/hotspot-integration-testing), the Wallet knows to use your approved Maker Wallet address. At this point, the Data Credits will be used for the transactions with your wallet address as the payer. 
```

## Hotspot Data Credit Fees

> In order for any Hotspot to join the Helium blockchain, its maker address must have sufficient Data Credits. A full list of [Helium blockchain data credit fees can be found here](/blockchain/transaction-fees). Hotspot Makers need to account for at least `5,000.000` total Data Credits per Hotspot. Specifically, for each Hotspot, a Maker must supply:

Reviewer 2: Capitalize Blockchain

Reviewer 2: Capitalize Maker

Author: Fixed.

* `4,000,000` Data Credits for the **Add Hotspot** fee. This is a one-time fee.
* `1,000,000` Data Credits for the **Assert Location** fee. Each Hotspot Maker is required to supply at least one **Assert Location* for their Hotspots. 

> As a maker, if you only the one-time **Add Hotspot** fee and one **Assert Location** fee, you'll supply `5,000.000` total Data Credits per Hotspot. Add `1,000,000` Data Credits for each additional **Assert Location** fee you with to supply.

Reviewer: should we add some notes about the transaction fees as well?

Author: Yes. Added.

:::warning 

If your wallet address does not have enough DCs to pay for required Hotspot Transactions, they will fail. 

:::

## Maker Integration Test Wallet Address

> During the course of your testing, you can use the Maker Integration Test wallet address created by the DeWi.  Here is that address:

Reviewer: ```suggestion
During the course of your testing, you can use the Maker Integration Test wallet address created by the DeWi.  This wallet does not contain any Data Credits so onboarding transactions will fail but it allows you to smoke test the onboarding steps. Here is that address:
```

> During the course of your testing, you can use the Maker Integration Test wallet address created by the DeWi.  Here is that address:

Reviewer 2: Add:

This Test wallet address can be used until you have your own designated Maker address. Adding Hotspots with this address as `payer` will not add Hotspots to the blockchain and you should not burn Data Credits into this address.

Author: Fixed.
